sandbox.html

```diff
@@ -6,22 +6,24 @@
     <title>Vue.js Study Jam</title>
 </head>
 <body>
-    <h1>Welcome to the Study Jam - How nice of you to join!!</h1>
+    <h1>Favourite Movie Series: Harry Potter</h1>
     <div id="app">
-        <h1 v-if="message.length % 2 == 0">{{ message }}</h1>
-        <p v-else>There is nothing</p>
-
-        <button @click="show = !show">Toggle List</button>
-        <button @click="list.push(list.length + 1)">Push Number</button>
-        <button @click="list.pop()">Pop Number</button>
-        <button @click="list.reverse()">Reverse List</button>
-
-        <ul v-if="show && list.length">
-            <li v-for="item of list">{{ item }}</li>
+        <h2>Characters</h2>
+        <ul>
+            <li v-for="(item, index) in list" :key="index">
+                {{ item }}
+                <button @click="addToFavorites(item)">Add to Favorites</button>
+            </li>
         </ul>
-
-  <p v-else-if="list.length">List is not empty, but hidden.</p>
-  <p v-else>List is empty.</p>
+        <h2>Favorites List</h2>
+        <ul v-if="favoriteList.length">
+            <li v-for="(item, index) in favoriteList" :key="index">{{ item }}</li>
+        </ul>
+        <form @submit.prevent="addItem">
+            <label for="newItem">Add new item:</label>
+            <input type="text" id="newItem" v-model="newItem">
```

Comment:
v-modal is a new way for two-way binding. It's majorly used for forms - It automatically handles what to look at and what to change. It's a handy directive.

[Here](https://v2.vuejs.org/v2/guide/components.html#Using-v-model-on-Components)

This is same as:
```vue
<input v-model="searchText">
```
as this:
```vue
<input
  v-bind:value="searchText"
  v-on:input="searchText = $event.target.value"
>
```